### ISSUE TITLE
Add ability to read / write data from / to VictoriaMetrics

### ISSUE THREAD

**Issue by valyala** [regel/loudml#250]
[VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics) is easy to operate and fast time series database. It would be great adding ability to read / write data from / to it. The following HTTP APIs can be used:

* [/api/v1/export](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md#how-to-export-time-series) for data reading
* [/api/v1/import](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md#how-to-import-time-series-data) or [/api/v1/import/csv](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md#how-to-import-csv-data) for data writing

**Comment by regel** [comment]
Hi Aliaksandr,

Contributions are welcome. You can open a new pull-request. Refer to the latest contributions including OpenTSDB and Prometheus (WIP) in the Pull requests tab for great examples.

Thanks

**Comment by vsergeyev** [comment]
Hello @valyala ,

From documentation my understanding VictoriaMetrics mimics Prometheus API in term of queries.
I'm currently working on https://github.com/regel/loudml/pull/254
This is Prometheus bucket support.

If it is a chance you may test it on your side and provide comments.

Regards,
V.

**Comment by valyala** [comment]
> From documentation my understanding VictoriaMetrics mimics Prometheus API in term of queries.

Yes. VictoriaMetrics supports Prometheus querying API - see [these docs](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md#prometheus-querying-api-usage). Additionally to this it supports `/api/v1/export` handler for returning raw data. See [this article](https://medium.com/@valyala/analyzing-prometheus-data-with-external-tools-5f3e5e147639) for the context.

> If it is a chance you may test it on your side and provide comments.

@dlazerka , could you look into https://github.com/regel/loudml/pull/254 and try using it with VictoriaMetrics?
